ssh/client.go

```diff
@@ -0,0 +1,153 @@
+package ssh
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
erm this library works quite poorly on windows and we did [some huge work](https://github.com/docker/docker/pulls?q=is%3Apr+ANSI+is%3Aclosed+created%3A%3E%3D2015-02-01+sort%3Acreated-asc) around terminal code in Docker 1.6 to port docker client to windows (and more will coming). 

Basically we need to emulate ANSI sequences on Windows (because cmd.exe/powershell doesn't do it). 

We should be importing `github.com/docker/docker/pkg/term` here instead for a better experience. Because even docker's Unix term package was initially ripped off/forked from this ssh/terminal but they diverged a lot over time as Docker changed it to accommodate the needs. This would also provide consistency across projects.